jme3-core/src/com/jme3/gde/core/util/datatransfer/TransformDataFromOriginal.java

```diff
@@ -0,0 +1,55 @@
+package com.jme3.gde.core.util.datatransfer;
+
+import com.jme3.gde.core.scene.ApplicationLogHandler;
+import com.jme3.gde.core.util.TaggedSpatialFinder;
+import com.jme3.scene.Geometry;
+import com.jme3.scene.Node;
+import com.jme3.scene.SceneGraphVisitorAdapter;
+import com.jme3.scene.Spatial;
+
+import java.util.logging.Logger;
+
+/**
+ * Copies Transform data (translation, rotation, scale) from an updated spatial
+ * to the original.
+ */
+public final class TransformDataFromOriginal implements 
+        SpatialDataTransferInterface {
+
+    private static final Logger LOGGER
+            = Logger.getLogger(AnimationDataFromOriginal.class.getName());
+
+    private final TaggedSpatialFinder finder;
+
+    public TransformDataFromOriginal(final TaggedSpatialFinder finder) {
+        this.finder = finder;
+    }
+
+    @Override
+    public void update(final Spatial root, final Spatial original) {
+        original.depthFirstTraversal(new SceneGraphVisitorAdapter() {
+
+            @Override
+            public void visit(final Geometry geom) {
+                final Geometry spat = (Geometry) finder.find(root, geom);
+                if (spat != null) {
+                    spat.setLocalTransform(geom.getLocalTransform());
+                    LOGGER.log(ApplicationLogHandler.LogLevel.FINE,
```

Comment:
Is this LogLevel import the right one?

Reply:
Not sure. It's used conservatively and I probably just copied this from SpatialUtil. That log is used to display the info bubbles and suck. But I think the SDK is too chatty with these things anyway, so it makes sense to remove it.